cmd/tink-cli/cmd/hardware/ip.go

```diff
@@ -33,15 +34,20 @@ var ipCmd = &cobra.Command{
 			if err != nil {
 				log.Fatal(err)
 			}
-			b, err := json.Marshal(hw)
-			if err != nil {
-				log.Fatal(err)
+			if hw.GetId() == "" {
+				log.Fatal("IP address not found in the database ", ip)
```

Comment:
It should be `log.Println()`. Please check for other occurances as well.

Reply:
But I would like to exit from here then it is better to use log.Fatal instead of log.Println().